### PR TITLE
[CDAP-5931] Tracker User Properties Management Part I

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-ui/app/features/tracker/controllers/metadata-ctrl.js [hunk 1]
```diff
@@ -153,9 +157,86 @@ class TrackerMetadataController{
...
+        this.myAlertOnValium.show({
+          type: 'danger',
+          content: err.data
+        });
+      });
+  }
+
+  propertyKeypress(event) {
+    switch (event.keyCode) {
+      case 13:
```
Comment (chtyim) [inline cdap-ui/app/features/tracker/controllers/metadata-ctrl.js]: Add a comment on what is 13 and 27.

File: cdap-ui/app/features/tracker/controllers/metadata-ctrl.js [hunk 2]
```diff
@@ -15,13 +15,17 @@
  */
 
 class TrackerMetadataController{
```
Comment (chtyim) [inline cdap-ui/app/features/tracker/controllers/metadata-ctrl.js]: should start to have the practice of documenting what a class is responsible for.

Comment (chtyim) [inline cdap-ui/app/features/tracker/controllers/metadata-ctrl.js]: Also, is it missing a space between the class name and the `{`?

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
demoed to @russorat

**Pull request merged**